### ISSUE TITLE
customer inspect with --output json returns table data

### ISSUE THREAD

**Issue by wyattwalter** [replicatedhq/replicated#463]
I have a script that relies on parsing data via `jq` from `replicated customer-inspect --customer <customer> --output json`. When I upgraded to 0.83.10 it stopped working. It was working from a release a month(?) or so ago, but I'm not sure the exact version I was using previously.

The output looks something like this:

```
replicated customer inspect --customer <name> --output json
ID: <id>
NAME: <name>
EMAIL: 
LICENSE_ID: <license_id>
EXPIRES: 2024-11-12 15:37:39 +0000 UTC


LOGIN:

Customer email required to login


INSTALL PREFLIGHT:

    curl https://krew.sh/preflight | bash


....
```

As you can see, the output is not json but is falling back to the default "table" data. 

`channel inspect` seems to work if I set `--output json` still, I haven't checked others.

**Comment by wyattwalter** [comment]
I worked around it for now by switching to:

```
replicated api get /v3/app/${APP_ID}/customers\?customerName\=${CUSTOMER_NAME}
```

**Comment by divolgin** [comment]
@wyattwalter this is now resolved in 0.83.11
